specs/ServerCertificate.md

```diff
@@ -0,0 +1,299 @@
+Server Certificate API
+===
+# Background
+
+The WebView2 team has been asked for an API to intercept when WebView2 cannot verify a server's digital certificate while loading a web page.
+This API provides an option to trust the server's TLS certificate at the application level and render the page without prompting the user about the TLS error or can cancel the request.
+
+# Description
+
+We propose adding `ReceivingServerCertificateError` API that allows you to verify TLS certificates with errors, and either continue the request
+to load the resource or cancel the request.
+
+When the event is raised, WebView2 will pass a `CoreWebView2ReceivingServerCertificateErrorEventArgs` , which lets you view the TLS certificate request Uri, error information, inspect the certificate metadata and several options for responding to the TLS request.
+
+* You can perform your own verification of the certificate and allow the navigation to proceed if you trust it.
+* You can choose to cancel the request.
+* You can choose to display the default TLS interstitial page to let user respond to the request.
+
+We also propose adding a `ClearServerCertificateErrorOverrideCache` API that clears certificate decisions that were cached in response to proceeding with TLS certificate errors.
+
+# Examples
+
+## Win32 C++
+``` cpp
+// When WebView2 doesn't trust a TLS certificate but host app does, this example bypasses
+// the default TLS interstitial page using the ReceivingServerCertificateError event handler and
+// continues the navigation to a server. Otherwise, cancel the request.
+void SettingsComponent::EnableCustomServerCertificateError()
+{
+  if (m_webview)
```

Comment:
Should be
```cpp
    auto webview11 = m_webview.query<ICoreWebView2_11>();
   if (webview11) { /* use webview11 instead of m_webview */ }
  else { FeatureNotAvailable(); }
```

(

Reply:
Make sense, will update.